gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/large_tests/platinum_partition_with_residual.json

```diff
@@ -0,0 +1,53 @@
+{
+  "test_name": "platinum-partition-with-residual",
+  "table_name": "platinum_partition_with_residual",
+  "input_pattern": "gs://genomics-public-data/platinum-genomes/vcf/*.vcf",
+  "partition_config": "gcp_variant_transforms/testing/data/partition_configs/integration_with_residual.yaml",
+  "runner": "DataflowRunner",
+  "variant_merge_strategy": "MOVE_TO_CALLS",
+  "worker_machine_type": "n1-highmem-4",
+  "max_num_workers": "40",
+  "num_workers": "40",
+  "assertion_configs": [
+    {
+      "query": ["NUM_ROWS_QUERY_chr01"],
```

Comment:
can you also add a query like:
```
SELECT COUNT(0) AS num_rows FROM `{TABLE_NAME}*`
```
to make sure there are no additional tables created. You can do the same for the no residual one as well to ensure the residual one is not created.

(you may need to have backquotes around `{TABLE_NAME}`.

Reply:
done.